recipe/bld.bat

```diff
@@ -44,3 +43,24 @@ for /f %%f in ('dir /b /S .\dist') do (
     pip install %%f
     if %ERRORLEVEL% neq 0 exit 1
 )
+
+REM After the build, check for the DLL
```

Comment:
cool, seems plausible - I think it would be good to make sure you're running tests that will access this .dll just to make sure everything's ok
also the dso_whitelist entry should then be removed

Reply:
oh wait hang on, I saw the previous commit that copied the dll to PREFIX/lib... that seems plausible perhaps

Reply:
Thanks! Yeah I will I just wanted to throw a curveball for the whitelist and see if it would play nice with it included (it didn't)